### ISSUE TITLE
"HTML5 attribute is in conflict with the associated WAI-ARIA attribute used on an input element" for TextArea

### ISSUE THREAD

**Issue by sjhowell** [carbon-design-system/carbon#8068]
## Brief description

"HTML5 attribute is in conflict with the associated WAI-ARIA attribute used on an input element" is reported by the "IBM Equal Access Accessibility Checker" plugin for a `<TextArea>` component with a `placeholder` attribute set.

## Environment

- Operating system: Windows and Mac
- Browser: Firefox and Chrome
- Automated testing tool and ruleset: IBM Accessibility Equal Access Toolkit
- Assistive technology used to verify: IBM Accessibility Equal Access Toolkit

## Detailed description

On our page we have a `TextArea` Carbon React component as follows:

```
<TextArea id="aiops_templateDescription"
    className=""
    value={templateDescription}
    placeholder={window.ASM_RESOURCES.TEMPLATE_DESCRIPTION_PLACEHOLDER}
    light={true}
    rows={1}
    hideLabel={false}
    invalid={false}
    labelText={window.ASM_RESOURCES.TEMPLATE_DESCRIPTION_LABEL}
    onChange={({target: {value}}) => setDescription(value)}
/>
```

The resulting HTML looks like this:

```
<textarea rows="1" cols="50" id="aiops_templateDescription" placeholder="Enter a description for your template" aria-placeholder="Enter a description for your template" class="bx--text-area bx--text-area--light"></textarea>
```

The IBM Accessibility Equal Access Toolkit raises the following violation for this:

![image](https://user-images.githubusercontent.com/40432849/110800018-aec61880-8273-11eb-9723-83307a09cd5c.png)
![image](https://user-images.githubusercontent.com/40432849/110800040-b5ed2680-8273-11eb-82ed-fba0ece3ce40.png)

```
HTML5 attributes must not conflict with the associated WAI-ARIA attribute used on an input element

Why is this important?

Using both an HTML5 attribute and the associated ARIA attribute with the same semantic meaning on an <input> element can cause a conflict that allows one of the attributes to override the other. This can lead to unexpected and confusing assistive technology behavior.

Element location

<textarea class="bx--text-area bx--text-area--light" aria-placeholder="Enter a description for your template" placeholder="Enter a description for your template" id="aiops_templateDescription" cols="50" rows="1">

What to do

Remove one of the conflicting HTML5 or ARIA attributes from the <input> element.
```

- What version of the Carbon Design System are you using? 7.25.0
- What did you expect to happen? No accessibilty violation to occur
- What happened instead? Accessibility violation occurred
- What WCAG 2.1 checkpoint does the issue violate? "3.3.2 Labels or Instructions"

## Steps to reproduce the issue

- Add a `TextArea` to your HTML page
- Set a placeholder attribute
- Render the page
- Observe that both a `placeholder` and `aria-placeholder` have been set

**Comment by emyarod** [comment]
this should be resolved by https://github.com/carbon-design-system/carbon/pull/7421

**Comment by sjhowell** [comment]
@emyarod That's good news - thanks for the info. I could not tell from 7421 though... what Carbon version is required to get that fix? Thanks

**Comment by emyarod** [comment]
that fix is contained in [v10.26.0](https://github.com/carbon-design-system/carbon/releases/tag/v10.26.0)

**Comment by sjhowell** [comment]
Thanks!
